api/src/main/java/dk/dbc/search/work/presentation/api/jpa/CacheEntity.java

```diff
@@ -0,0 +1,166 @@
+/*
```

Comment:
Hvorfor bruger vi en cache i databasen, istedet for f.eks. en Redis cache (eller den der, der ligger i Payara)?

Reply:
Fordi den er "meget" permanent. (uendelig levetid med forceret update når datoen i corepo ændrer sig)